prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/CO/co-phd.yml

```diff
@@ -0,0 +1,224 @@
+- name: "co-phd"
+  description: "Colorado Department of Public Health and Environment"
+  jurisdiction: "STATE"
+  stateCode: "CO"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+  - name: "full-elr-secondary"
+    organizationName: "co-phd"
+    topic: "full-elr"
+    customerStatus: "active"
+    translation: !<HL7>
+      schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/CO/CO-receiver-transform.yml"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "CO-ELR"
+      receivingApplicationOID: null
+      receivingFacilityName: "CO"
+      receivingFacilityOID: null
+      messageProfileId: null
+      replaceValue: {}
+      replaceValueAwithB:
+        MSH-6:
+        - '*': "CO"
+        MSH-5:
+        - '*': "CO-ELR"
+        MSH-3:
+        - '*': "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO"
+      reportingFacilityName: "CDC PRIME"
```

Comment:
We don't need values for reportingFacilityName and reportingFacilityId any more since we handle them in receiver transform.

Reply:
These are still needed for creating the batch headers. If you have been removing them, we need to put them back otherwise the FHS and BHS segments won't match the MSH segments